Uplift/Views/Home/FitnessCenterView.swift

```diff
@@ -16,6 +16,17 @@ struct FitnessCenterView: View {
     let fc: Facility?
 
     @StateObject private var viewModel = ViewModel()
+    @State private var fetchedEqmt: [Equipment] = Equipment.dummyData
+    /// Dictionary to track the number of equipment objects in each EquipmentType category
+    @State private var eqpmtTypeDict: [EquipmentType: Int] = [
+        .cardio: 0,
+        .freeWeights: 0,
+        .miscellaneous: 0,
+        .multiCable: 0,
+        .plateLoaded: 0,
+        .racksAndBenches: 0,
+        .selectorized: 0
+    ]
```

Comment:
One thing I would like to add is that not everything needs an `@State` property wrapper. Mutability is not always a good thing.